From 1bfb12d081e84019caafc32cad279b988023f0c7 Mon Sep 17 00:00:00 2001
From: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
Date: Wed, 29 Sep 2021 23:44:41 +0300
Subject: [PATCH 149/322] Revert "[FIO extras] imx8mm_evk: add default bootcmd
 to defconfig"

This reverts commit 2943edd98f3029e432f9c8806f31fec7cbd86ce6.

The CONFIG_BOOTCOMMAND is introduced in defconfig in the
commit 85e00a89b6 ("MLK-23574-27 imx8mm_evk: Update board codes and add DDR4 support")
Remove duplicated option assigning.

Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
---
 configs/imx8mm_evk_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configs/imx8mm_evk_defconfig b/configs/imx8mm_evk_defconfig
index 9ea250310e2..1ada21f4455 100644
--- a/configs/imx8mm_evk_defconfig
+++ b/configs/imx8mm_evk_defconfig
@@ -97,7 +97,6 @@ CONFIG_FASTBOOT_UUU_SUPPORT=y
 CONFIG_FASTBOOT_BUF_ADDR=0x42800000
 CONFIG_FASTBOOT_BUF_SIZE=0x40000000
 CONFIG_FASTBOOT_FLASH=y
-CONFIG_BOOTCOMMAND="mmc dev ${mmcdev}; if mmc rescan; then if run loadbootscript; then run bootscript; else if run loadimage; then run mmcboot; else run netboot; fi; fi; fi;"
 
 CONFIG_DM_I2C=y
 CONFIG_SYS_I2C_MXC=y
-- 
2.34.1

